### PR TITLE
CAD-3928 minor polish

### PR DESCRIPTION
**Pull request by NeilBurgess42**
Minor changes to conform with IOG style guide.

### CODE REVIEW

**Review by Jimbo4350 — APPROVED** [review]

**Review by olgahryniuk — APPROVED** [review]

File: doc/reference/cardano-node-cli-reference.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ The command line interface (CLI) provides a collection of tools for generating k
 This section provides a reference of the core `cardano-cli` commands and their associated subcommands:
 
 *cardano-cli*
-The set of `cardano-cli` commands include:
+The set of `cardano-cli` commands includes:
```
Comment (newhoggy) [inline doc/reference/cardano-node-cli-reference.md:8]: I think `include` is the right grammar here.

Comment (NeilBurgess42) [reply doc/reference/cardano-node-cli-reference.md:8]: No, 'The set... includes' is correct.

**Pull request merged**